include/Text.hpp

```diff
@@ -0,0 +1,157 @@
+#ifndef RAYLIB_CPP_INCLUDE_TEXT_HPP_
+#define RAYLIB_CPP_INCLUDE_TEXT_HPP_
+
+#include <string>
+
+#include "./raylib.hpp"
+#include "./RaylibException.hpp"
+#include "./raylib-cpp-utils.hpp"
+
+namespace raylib {
+/**
+ * Text Functions.
+ */
+class Text {
+ public:
+    /**
+     * Initializes a new Text object.
+     * @param text Text to initialize.
+     * @return void.
+     */
+    Text(const std::string& text = "")
+    {
+        SetText(text);
+    }
+    
+    /**
+     * Initializes a new Text object.
+     * @param text Text to initialize.
+     * @param posX X position of the text.
+     * @param posY Y position of the text.
+     * @return void.
+     */
+    Text(const std::string& text, const float posX, const float posY)
+    {
+        _text = text;
+        _position.x = posX;
+        _position.y = posY;
```

Comment:
Initializer lists for the constructors may be a good call here https://en.cppreference.com/w/cpp/language/constructor#Initialization_order